### ISSUE TITLE
Make the type case insensitive

### ISSUE THREAD

**Issue by Heziode** [yepengding/bibtex-js-parser#1]
Currently, type is sensitive, which is not convenient.

For example, the following example does not work:

```
@InProceedings{Foo,
  author    = {Tom and the gang},
  booktitle = {Something amazing},
  title     = {An awesome research},
  year      = {2022},
  month     = {jun},
}
```

But with type all lowercase, it works:

```
@inproceedings{Foo,
  author    = {Tom and the gang},
  booktitle = {Something amazing},
  title     = {An awesome research},
  year      = {2022},
  month     = {jun},
}
```

**Comment by yepengding** [comment]
Thanks for your attention. This will be fixed in the next version.

**Comment by yepengding** [comment]
The case-insensitive entry type support has been included in the version [1.1.6](https://www.npmjs.com/package/bibtex-js-parser/v/1.1.6).
Thank you for your feature suggestion. @Heziode
